x-pack/apm-server/main.go

```diff
@@ -150,6 +151,19 @@ func newProcessors(args beater.ServerParams) ([]namedProcessor, error) {
 			return nil, errors.Wrapf(err, "error creating %s", serviceName)
 		}
 		processors = append(processors, namedProcessor{name: serviceName, processor: serviceAggregator})
+
+		const serviceSummaryName = "service summary aggregation"
+		args.Logger.Infof("creating %s with config: %+v", serviceSummaryName, args.Config.Aggregation.Service)
+		serviceSummaryAggregator, err := servicesummarymetrics.NewAggregator(servicesummarymetrics.AggregatorConfig{
+			BatchProcessor:  args.BatchProcessor,
+			Interval:        metricsInterval,
+			RollUpIntervals: rollUpMetricsIntervals,
+			MaxGroups:       args.Config.Aggregation.Service.MaxGroups,
+		})
```

Comment:
Reusing service metrics config `enabled` and `max_groups` here. Is this fine or should we introduce a new set of config for service summary?

Reply:
clarified on the issue - fine to reuse for now